Wikipedia/Code/SectionEditorWebViewConfiguration.swift

```diff
@@ -1,7 +1,10 @@
 import WebKit
 
 protocol SectionEditorWebViewSelectionChangedDelegate: NSObjectProtocol {
-    func turnOffAllButtonHighlights()
+    // Inside 'selectionChanged' the delegate should de-select all buttons.
+    func selectionChanged(isRangeSelected: Bool)
```

Comment:
@nambatee I changed the name of this one passing it the `isRangeSelected` value you needed.